org-formation/100-shared-dns/_tasks.yaml

```diff
@@ -0,0 +1,54 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+SagebioAppZone:  # Zone for application front-end DNS names, e.g. 'dca.app.sagebionetworks.org'
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/R53-hostedzone.yaml
+  StackName: !Sub '${resourcePrefix}-sagebio-app-zone'
+  StackDescription: Create a shared hosted zone for application domains
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref SageITAccount
+  Parameters:
+    DnsDomainName: "app.sagebionetworks.org"
+
+SagebioAppZoneAcmCertificate:  # Wildcard certificate for '*.app.sagebionetworks.org'
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/acm-certificate.yaml
+  StackName: !Sub '${resourcePrefix}-sagebio-app-cert'
+  StackDescription: Create a wildcard certificate for application domains
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    # We need a copy of this certificate in every account where it will be used
+    Account:
+      - !Ref DnTDevAccount
+      - !Ref DCAProdAccount
+      - !Ref GenieProdAccount
```

Comment:
creating a cert requires manual validation and this is doing it for 3 accounts.  what is your plan to validate across all these accounts?

Reply:
my plan is to manually validate these certs in the portal after merging this PR and before updating the CDK deploy app to use the cert